index.html

```diff
@@ -29,16 +30,18 @@
           <h2 class="primary-text">I'm Juan Diaz Glad to see you!</h2>
```

Comment:
- [ ] Consider the heading hierarchy is meaningful. Ideally, the page starts with an "h1" , so this should be an `h1` instead of `h2`

Reply:
Done! thanks for the feedback!